libbeat/docs/command-reference.asciidoc

```diff
@@ -752,7 +752,8 @@ endif::no_dashboards[]
 
 ifdef::has_ml_jobs[]
 * The machine learning jobs contain the configuration information and metadata
-necessary to analyze data for anomalies.
+necessary to analyze data for anomalies. You can only load the jobs to Elasticsearch
```

Comment:
This statement is a little ambiguous. It might be better not to mention 7.x and just say,  "Starting in version 8.0.0, this command is no longer supported. To load machine learning jobs, use the Machine Learning UI in Kibana."

To ruflin's question, I don't see a great landing page in the Machine Learning docs. We could send users to the Kibana docs, but then they need to play hopscotch over to the Machine Learning docs. @lcawl Where should this link point?